src/main/flight/servos.c

```diff
@@ -296,20 +296,54 @@ STATIC_UNIT_TESTED void forwardAuxChannelsToServos(uint8_t firstServoIndex)
     }
 }
 
+static void biquadFilterInitd1d2(biquadFilter_t *filter, float sample)
```

Comment:
@ellingo so basically this function "resets" the biquad filter. For consistency with PT1 filter this function can be named `biquadFilterReset` and moved to `filter.c`.